### PR TITLE
Cross platform submission

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

File: estimation/model.py [hunk 1]
```diff
@@ -225,7 +225,7 @@ def save_model(self, output_file: str):
 
         self._model.save(output_file)
 
-        with h5py.File(output_file, "r+") as f:
+        with h5py.File(output_file, "r") as f:
```
Comment (jasonmotha) [inline estimation/model.py]: Is this a merge issue?

Comment (claudio525) [reply estimation/model.py]: This change is incorrect, this is the save function so it requires write access. So this change should be reverted.

File: scripts/cybershake/auto_submit.py [hunk 2]
```diff
@@ -21,6 +21,15 @@
 default_n_runs = 12
 default_1d_mod = "/nesi/transit/nesi00213/VelocityModel/Mod-1D/Cant1D_v2-midQ_leer.1d"
 
+job_run_machine = {
+    const.ProcessType.EMOD3D.value: "maui",
+    const.ProcessType.merge_ts.value: "mahuika",
+    const.ProcessType.winbin_aio.value: "maui",
+    const.ProcessType.HF.value: "maui",
+    const.ProcessType.BB.value: "maui",
+    const.ProcessType.IM_calculation.value: "maui",
```
Comment (claudio525) [inline scripts/cybershake/auto_submit.py]: In qcore constants there is an enum for the different HPC systems, might be worth using that

File: scripts/submit_sim_imcalc.py [hunk 3]
```diff
@@ -6,9 +6,10 @@
 
 import qcore.constants as const
 from qcore import utils, shared
+from qcore.config import host
 from typing import Dict
 from estimation.estimate_wct import est_IM_chours_single
-from shared_workflow.shared import submit_sl_script, set_wct, confirm, resolve_header
+from shared_workflow.shared import submit_sl_script, set_wct, confirm, get_partition, resolve_header
```
Comment (claudio525) [inline scripts/submit_sim_imcalc.py]: get_partition doesn't appear to be used?

**Pull request merged**